src/consent-state.js

```diff
@@ -29,3 +29,18 @@ export const CONSENT_POLICY_STATE = {
   UNKNOWN_NOT_REQUIRED: 3,
   UNKNOWN: 4,
 };
+
+/**
+ * Defines valid consent string types passed
+ * in by consent iframe metadata.
+ * @enum {number}
+ */
+export const CONSENT_STRING_TYPE = {
+  // Enum value has external dependency for metadata collection.
+  // Please do not change existing value.
+  // If new values are added, please notify the AMP for Ads team to assure
```

Comment:
AMP can add support to new consent string type, without ads team's support. I can see us adding new values without affecting the RTC behavior. 
One thing worth mention is that it's up to vendors to interpret the consent string if the consent string type is undefined.

Reply:
Thanks for the additions :)